source/reference/minio-server/settings/console.rst

```diff
@@ -176,6 +178,148 @@ Log Query URL
       This setting does not have a configuration variable setting.
       Use the Environment Variable instead.
 
+Content Security Policy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+*Optional*
+
+Set the `Content-Security-Policy <https://en.wikipedia.org/wiki/Content_Security_Policy>`__ header in MinIO Console responses.
+Defaults to ``default-src 'self' 'unsafe-eval' 'unsafe-inline';``
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_BROWSER_CONTENT_SECURITY_POLICY
+
+      .. code-block:: shell
+         :class: copyable
+
+         set MINIO_BROWSER_CONTENT_SECURITY_POLICY="default-src 'self' 'unsafe-eval' 'unsafe-inline';"
+
+   .. tab-item:: Configuration Setting
+      :sync: config
+
+      .. mc-conf:: browser csp_policy
+         :delimiter: " "
+
+      .. code-block:: shell
+
+         mc admin config set browser \
+            csp_policy="default-src 'self' 'unsafe-eval' 'unsafe-inline';" \
+            [ARGUMENT=VALUE ...]
```

Comment:
Is this the correct syntax?

Reply:
the double/single quotes certainly opens up some space of typos.

https://github.com/minio/minio/pull/18631/files#diff-36cb1e40f5d5539119315861c1595565234347c49b0d61a4861ecb1b5efd97bdR54

But that does look like it.